processor-api/src/main/java/com/synopsys/integration/alert/processor/api/NotificationProcessor.java

```diff
@@ -55,19 +62,33 @@ public NotificationProcessor(
 
     public void processNotifications(List<AlertNotificationModel> notifications, List<FrequencyType> frequencies) {
         try {
+            if (logger.isDebugEnabled()) {
+                List<Long> notificationIds = notifications.stream()
+                                                 .map(AlertNotificationModel::getId)
+                                                 .collect(Collectors.toList());
+                String joinedIds = StringUtils.join(notificationIds, ", ");
+                notificationLogger.debug("Processing notifications: {}", joinedIds);
+            }
             processAndDistribute(notifications, frequencies);
             notificationAccessor.setNotificationsProcessed(notifications);
+            if (logger.isDebugEnabled()) {
+                List<Long> notificationIds = notifications.stream()
+                                                 .map(AlertNotificationModel::getId)
+                                                 .collect(Collectors.toList());
+                String joinedIds = StringUtils.join(notificationIds, ", ");
+                notificationLogger.debug("These notifications have been processed: {}", joinedIds);
```

Comment:
The second part of the pair of messages to make it easier to correlate the start and finish.
```suggestion
                notificationLogger.debug("Finished processing notifications: {}", joinedIds);
```